pages/home/glossary.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: "Glossary"
+description: "Glossary of terms used in the Arcade platform"
+---
+
+# Arcade Glossary
+
+
+## Agents and Tools
+
+```mermaid
+graph TD
+    A[Agent] --> B[Toolkit]
+    B --> C[Tool]
+```
+
+### Agent
+
+An 'agent' is the application you are building.  It can be a chatbot, a web application, a mobile app, or any other type of application that happens to use an LLM to as part of its functionality.  Agents interact with the world by calling tools.  Helping you build, test, authenticate, and deploy tools is what Arcade is all about.
+
+### Toolkit
+
+A 'toolkit' is a collection of tools that can be used by an agent, grouped logically together by a common theme or provider.  Toolkits are the unit of deployment for tools within Arcade.
+
+### Tool
+
+A 'tool' is a function that can be called by an agent which performs some action - commonly via an API, filesystem, database, etc.  Tools are written in python and deployed by running a worker which contains the toolkit's code.  Tools are defined by the `@tool()` decorator and will be passed `ToolContext` as the first argument.  If a tool has dependencies that are not met (a secret is not provided, for example), the tool will fail to execute.
+
+Tools are commonly referred to by a qualified name that includes their toolkit. For example, [Gmail.SendEmail](https://docs.arcade.dev/toolkits/productivity/gmail#gmailsendemail)
+
+*Learn more about [tools](/home/build-tools/create-a-toolkit).*
+
+### Tool Context
+
+'Tool context' is an object that is passed to a tool as a parameter when the tool is executed.  It contains information about the tool call, the user for which the tool is being called, and any secrets the tool requires to run.
+
+*Learn more about [tool context](/home/build-tools/tool-context).*
+
+### MCP (Model Context Protocol)
+
+MCP is an open standard protocol that enables LLMs to access and use tools and data sources. Arcade tools go further than MCP with enterprise-grade authentication, secure token management, and fine-grained permissions. This allows your tools to be accessible to any LLM that supports MCP, while maintaining security and user privacy.
+
+Arcade Tools can be presented as an MCP server (via the Arcade Engine), allowing any LLM that supports MCP to access your tools.  In the future, remote MCP servers will be able to be added to your project as additional workers, making their tools available to your agents.
+
+*Learn more about [MCP and Arcade](/home/mcp-overview).*
+
+### OXP
+
+OXP is Arcade's proprietary protocol for tool definition and tool calling that enables AI/LLM agents to access tools and services.  OXP is a more secure and performant alternative to MCP, and is the preferred way to access tools from your agents.
```

Comment:
Is this proprietary? If so, let's consider renaming it so people that already know about https://github.com/OpenExecProtocol/Specification don't complain